OptionSelector/CQShadesData.m

```diff
@@ -0,0 +1,220 @@
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "CQShadesData.h"
+#import <UIKit/UIKit.h>
+
+@implementation CQShadesData
+
+-(void)initializeData{
+
+    self.red = [[CQCategory alloc] init];
+    self.yellow = [[CQCategory alloc] init];
+    self.blue = [[CQCategory alloc] init];
+    
+    self.red.name = @"Red";
+    self.yellow.name = @"Yellow";
+    self.blue.name = @"Blue";
+    
+    self.red.detailTitle = @"Shades of Red";
+    self.yellow.detailTitle = @"Shades of Yellow";
+    self.blue.detailTitle = @"Shades of Blue";
+    
+    self.red.options = @[@"cherry",
+                         @"rose",
+                         @"jam",
+                         @"merlot",
+                         @"garnet",
+                         @"crimson",
+                         @"ruby",
+                         @"scarlet",
+                         @"wine",
+                         @"brick",
+                         @"apple",
+                         @"mahogany",
+                         @"blood",
+                         @"sangria",
+                         @"berry",
+                         @"currant",
+                         @"blush",
+                         @"candy",
+                         @"lipstick"
+                         ];
+    
+    self.yellow.options = @[@"canary",
+                            @"gold",
+                            @"daffodil",
+                            @"flaxen",
+                            @"butter",
+                            @"lemon",
+                            @"mustard",
+                            @"corn",
+                            @"medallion",
+                            @"dandelion",
+                            @"fire",
+                            @"bumblebee",
+                            @"banana",
+                            @"butterscotch",
+                            @"dijon",
+                            @"honey",
+                            @"blonde",
+                            @"pineapple",
+                            @"tuscan sun"
+                            ];
+    
+    self.blue.options = @[@"slate",
+                          @"sky",
+                          @"navy",
+                          @"indigo",
+                          @"cobalt",
+                          @"teal",
+                          @"ocean",
+                          @"peacock",
+                          @"azure",
+                          @"cerulean",
+                          @"lapis",
+                          @"spruce",
+                          @"stone",
+                          @"aegean",
+                          @"blueberry",
+                          @"denim",
+                          @"admiral",
+                          @"sapphire",
+                          @"arctic"
+                          ];
+   
+    //Without (CGFLoat) OR the decimal (.0) after the number the UIColor will not show; one OR the other is sufficient.
```

Comment:
In C (and thus objective C), unless a number has a decimal in it (or is cast to a floating point type), it's assumed to be an integer. Also, all of the basic operators return a number of the same type as their operands. This is a problem for division because it means that the number is truncated, so 164/255, since it's less than 1, just becomes 0. Putting the .0 on the denominator or numerator should fix it (like you figured out).